concrete-core-fixture/src/fixture/lwe_ciphertext_vector_zero_encryption.rs

```diff
@@ -0,0 +1,167 @@
+                    lwe_dimension: LweDimension(1000),
+                    lwe_ciphertext_count: LweCiphertextCount(100),
+                },
+                LweCiphertextVectorZeroEncryptionParameters {
+                    noise: Variance(0.00000001),
+                    lwe_dimension: LweDimension(3000),
+                    lwe_ciphertext_count: LweCiphertextCount(100),
+                },
+                LweCiphertextVectorZeroEncryptionParameters {
+                    noise: Variance(0.00000001),
+                    lwe_dimension: LweDimension(6000),
+                    lwe_ciphertext_count: LweCiphertextCount(100),
+                },
+            ]
+            .into_iter(),
+        )
+    }
+
+    fn generate_random_repetition_prototypes(
+        parameters: &Self::Parameters,
+        maker: &mut Maker,
+    ) -> Self::RepetitionPrototypes {
+        let proto_secret_key = maker.new_lwe_secret_key(parameters.lwe_dimension);
+        (proto_secret_key,)
+    }
+
+    fn generate_random_sample_prototypes(
+        _parameters: &Self::Parameters,
+        _maker: &mut Maker,
+        _repetition_proto: &Self::RepetitionPrototypes,
+    ) -> Self::SamplePrototypes {
+    }
+
+    fn prepare_context(
+        _parameters: &Self::Parameters,
+        maker: &mut Maker,
+        repetition_proto: &Self::RepetitionPrototypes,
+        _sample_proto: &Self::SamplePrototypes,
+    ) -> Self::PreExecutionContext {
+        let (proto_secret_key,) = repetition_proto;
+        maker.synthesize_lwe_secret_key(proto_secret_key)
+    }
+
+    fn execute_engine(
+        parameters: &Self::Parameters,
+        engine: &mut Engine,
+        context: Self::PreExecutionContext,
+    ) -> Self::PostExecutionContext {
+        let secret_key = context;
+        let ciphertext_vector = unsafe {
+            engine.zero_encrypt_lwe_ciphertext_vector_unchecked(
+                &secret_key,
+                parameters.noise,
+                parameters.lwe_ciphertext_count,
+            )
+        };
+        (secret_key, ciphertext_vector)
+    }
+
+    fn process_context(
+        parameters: &Self::Parameters,
+        maker: &mut Maker,
+        repetition_proto: &Self::RepetitionPrototypes,
+        _sample_proto: &Self::SamplePrototypes,
+        context: Self::PostExecutionContext,
+    ) -> Self::Outcome {
+        let (secret_key, ciphertext_vector) = context;
+        let (proto_secret_key,) = repetition_proto;
+        let proto_output_ciphertext_vector =
+            maker.unsynthesize_lwe_ciphertext_vector(&ciphertext_vector);
+        maker.destroy_lwe_ciphertext_vector(ciphertext_vector);
+        maker.destroy_lwe_secret_key(secret_key);
+        let proto_plaintext = maker.decrypt_lwe_ciphertext_vector_to_plaintext_vector(
+            proto_secret_key,
+            &proto_output_ciphertext_vector,
+        );
+        (
+            vec![Precision::Raw::ZERO; parameters.lwe_ciphertext_count.0],
```

Comment:
Here you could have also used `Precision::Raw::zero_vec(parameters.lwe_ciphertext_count.0)`